.vale/styles/Infrahub/branded-terms-case-swap.yml

```diff
@@ -10,7 +10,7 @@ swap:
   (?:[^"]docker): Docker
   (?:[Dd]ockerhub): DockerHub
   (?:[^/][Gg]ithub): GitHub
-  (?:[Gg]itlab): GitLab
```

Comment:
I don't know why this PR is hitting these docs issues on unmodified file `repository.mdx`. Anyway, I modified GitLab pattern so it passes `#gitlab-access-token` occurences.